omim2obo/main.py

```diff
@@ -280,49 +279,45 @@ def omim2obo(use_cache: bool = False):
-            add_subclassof_restriction_with_evidence(graph, predicate, OMIM[p_mim], OMIM[mim_number], evidence)
-            if predicate in MORBIDMAP_PHENOTYPE_MAPPING_KEY_INVERSE_PREDICATES:
-                # The following code basically just adds the converse relation:
-                # If there is g2d in the previous code, we now, in addition, add d2g
-                inverse_predicate = MORBIDMAP_PHENOTYPE_MAPPING_KEY_INVERSE_PREDICATES[predicate]
-                add_subclassof_restriction_with_evidence(
-                    graph, inverse_predicate, OMIM[mim_number], OMIM[p_mim], evidence)
+    gene_phenotypes: Dict[str, Dict] = parse_morbid_map(get_mim_file('morbidmap', download_files_tf))
+
+    # - Cyto location: Add RO:0002525 (is subsequence of)
+    # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
+    for gene_mim, gene_data in gene_phenotypes.items():
+        if gene_data['cyto_location']:
+            chr_id = '9606chr' + gene_data['cyto_location']  # todo: document meaning of 9606chr
+            add_subclassof_restriction(graph, RO['0002525'], CHR[chr_id], OMIM[gene_mim])
+
+    # TODO: @Trish: I've removed addition of gene->disease relations, leaving only disease->gene. Is that what we want?
+    #  Removed because extra complexity. Want to get down disease->gene code first. And Mondo doesn't need them.
+
+    # - Disease-Gene relations, 1 of 2: Collect phenotype MIMs & associated gene MIMs and relationship info
+    phenotype_genes: Dict[str, List[Dict[str, str]]] = defaultdict(list)
+    for gene_mim, gene_data in gene_phenotypes.items():
+        for assoc in gene_data['phenotype_associations']:
+            p_mim, p_lab, p_map_key, p_map_lab = assoc['phenotype_mim_number'], assoc['phenotype_label'], \
+                assoc['phenotype_mapping_info_key'], assoc['phenotype_mapping_info_label']
+            if not p_mim:  # not an association to another MIM; ignore
+                continue  # see: https://github.com/monarch-initiative/omim/issues/78
+            # TODO: @Trish: Should we still consider this? if so, here or further down?
+            # if p_map_key == '1':  # association w/ unknown defect; skip
+            #     continue  # See: https://github.com/monarch-initiative/omim/issues/79#issuecomment-1319408780)
+            phenotype_genes[p_mim].append({
+                'gene_id': gene_mim, 'phenotype_label': p_lab, 'mapping_key': p_map_key, 'mapping_label': p_map_lab})
+
+    # - Disease-Gene relations, 2 of 2: Add relations (subclass restrictions)
+    #   Only add if (1) only 1 association (2)‚mapping key = 3, (3) 'definitive'
+    phenotype_genes2: Dict[str, Dict[str, str]] = {k: v[0] for k, v in phenotype_genes.items() if len(v) == 1}  # 1assoc
+    phenotype_genes2 = {k: v for k, v in phenotype_genes2.items() if p2g_is_definitive(v['phenotype_label'])}
+    for p_mim, assoc in phenotype_genes2.items():
```

Comment:
I believe this will still include the handful of diseases where [p_label contains the word 'digenic'](https://docs.google.com/document/d/1cLfBgPIZWiN5LX-E-xwSyBeFdT-vw0JuSfSM7HL3_hc/edit?tab=t.0#heading=h.h4y343h64cck) since there are cases where these exist with 1 gene association and a mapping key of 3 and are not prefixed with an omim special character.

Reply:
Yes, that's true. I was going to follow up with you about that.

Basically my rationale for not filtering these out is because of the OMIM team's response. It seems to me that they haven't really insured the data quality of digenic diseases. So I suppose that could mean one of the following or both:
1. Non digenic, but being labeled as such
2. Digenic but they only labeled one of the associations

I had the first case at the forefront of my mind. But now that I think about it, for these cases where it is labeled diegenic but it only has one association, we have no idea which of these two cases it is. 

So therefore, I do agree it is a good idea to filter them out now. 

However I wonder if we want to consider outputting them into a special table for curator review. Perhaps some of them are not actually digenic?

Reply:
Sure, I think outputting them to a file for curator review is a good idea!

Reply:
Done!
I added `review.tsv` to the release files and documentation for it, including the types of cases found inside of it, in the `README.md`.

- [`review.tsv` Google Sheet](https://docs.google.com/spreadsheets/d/1ftVMjsAtzVLv4aZFQ7KelRfFjct4VEprn6E76EdDJgw/edit?gid=441961077#gid=441961077)